cmd/kube-scheduler/app/options/options.go

```diff
@@ -19,6 +19,7 @@ package options
 
 import (
 	"fmt"
+	"k8s.io/client-go/util/clientutil"
```

Comment:
nit: better place this lib in below k8s.io section

Reply:
expected go-fmt-ed changes